desktop/core/src/desktop/js/jquery/plugins/jquery.filechooser.js

```diff
@@ -834,6 +834,43 @@ function initUploader(path, _parent, el, labels) {
       </div>
     `;
     document.body.appendChild(qqTemplate);
+
+    function pollForTaskProgress(taskId, listItem, fileName) {
+      let taskStatus = 'pending';
+      const pollingInterval = 10000; // 10 seconds
+
+      const doPoll = function () {
+        if (taskStatus === 'pending') {
+          $.get('/desktop/api2/taskserver/check_upload_status/' + taskId, data => {
+            if (data.isFinalized || data.isFailure || data.is_revoked) {
```

Comment:
you have some camelCase and some snake_case here. Is that something easy to clean up?